lib/recommendation.js

```diff
@@ -0,0 +1,61 @@
+var XMLHttpRequest = Cc['@mozilla.org/xmlextras/xmlhttprequest;1'].createInstance();
+
+
+class RecommendationServer {
```

Comment:
Note also that FF only enabled `class` in FF 45 ([mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), [tracking bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1197932)), and 45 doesn't go GA until 3/8/15. It might get backed out, and I'm not sure about adding a transpiler to this repo (though we could talk about it in a discussion bug, if you want to file one).